Auctionator.toc

```diff
@@ -21,7 +21,6 @@ Locales\Manifest.xml
 Source\Manifest.xml
 
 AuctionatorUtil.lua
-AuctionatorFilters.lua
```

Comment:
Can we remove this file, too?

Reply:
I've just moved AuctionatorFilters.lua to Source/Search/Filters.lua as we still need the stuff it generates :)

Reply:
I think Auctionator.toc might be necessary, I'm not sure